### PR TITLE
[IMP] web,*: replace ClipboardJS with Web API and homebrew polyfill

### PR DESCRIPTION
**Pull request by pparidans**
Implements missing `clipboard.write()` and `ClipboardItem` ; targets
only Firefox (which partially implements Clipboard API, features hidden
behind a preference flag).

Supported browsers: Chromium 76+, Safari 13.1+, Firefox 63+, Safari iOS
13.4+.

This allows to replace the ClipboardJS by the native Clipboard API.

Reference:
https://caniuse.com/?search=clipboard
https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API
https://w3c.github.io/clipboard-apis/

task-3439226

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pparidans** [comment]
@odoo/rd-security The incriminated code is basically a move of the said line. If there is a better way of writing this, please let me know :slightly_smiling_face:

File: addons/web/static/src/libs/clipboard_polyfill.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/** @odoo-module **/
+
+class ClipboardItemImpl {
```
Comment (aab-odoo) [inline addons/web/static/src/libs/clipboard_polyfill.js:3]: I know @ged-odoo doesn't like the libs/ folder. I'm wondering where we would put this file if we didn't have that folder. It's not a lib. It exports nothing so it doesn't fit well in the core folder. And we do no want to see to many root files in the web/

**Review by aab-odoo — APPROVED** [review]

File: addons/web/static/src/libs/clipboard_polyfill.js (hunk 1 above)
Comment (ged-odoo) [reply addons/web/static/src/libs/clipboard_polyfill.js:3]: well in this case, it is not a lib, it is a polyfill. I would put it in `src/polyfills/clipboard.js` I guess. Not a big fan, but at least it's not lying

**Review by adr-odoo — APPROVED** [review]
@robodoo r+

**Review by adr-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**